sdk/cosmosdb/cosmos/src/utils/encode.ts

```diff
@@ -5,3 +5,17 @@ export function encodeUTF8(str: string): Uint8Array {
   }
   return bytes;
 }
+
+export function encodeBase64(value: ArrayBuffer): string {
```

Comment:
This may be something we want in a core library since core-http does something similar

Reply:
+1 to this, but we can factor this out at a later date I assume.